akvo/iati/imports/mappers/defaults.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+from ....rsr.models.project import Project
+from ....rsr.models.project_condition import ProjectCondition
+
+from .. import ImportMapper
+
+CODE_TO_STATUS = {
+    '1': 'H',
+    '2': 'A',
+    '3': 'C',
+    '4': 'C',
+    '5': 'L',
+    '6': 'R'
+}
+
+
+class Language(ImportMapper):
+
+    def do_import(self):
+        """
+        Retrieve and store the language.
+        The language will be extracted from the 'lang' attribute of the activity root element.
+
+        :return: List; contains fields that have changed
+        """
+
+        xml_ns = 'http://www.w3.org/XML/1998/namespace'
```

Comment:
Maybe we can create a function similar to ```akvo_ns(tag)``` for ```xml_ns(tag)```? Even though it's only used twice for now.